Admin/AbstractAdmin.php

```diff
@@ -422,8 +422,6 @@
      */
     protected $securityInformation = array();
 
-    protected $cacheIsGranted = array();
```

Comment:
I mean will always be empty, never read or write by sonata, so it may break classes extending AbstractAdmin, no?